copy/tools/chartist.yml

```diff
@@ -0,0 +1,54 @@
+title: Chartist
+description: Simple responsive SVG charts
+logo: chartist.svg
+developer: null
+based_on: null
+licenses:
+  - type: open-source
+    title: MIT
+    link: 'https://github.com/chartist-js/chartist/blob/main/LICENSE-MIT'
```

Comment:
There are two licenses

Reply:
If I add another license, only the last one is displayed. Which one should it be, MIT or WTFPL?

Reply:
wtfpl is more fun )

Reply:
Fixed : )